site/phpstan-baseline.neon

```diff
@@ -60,11 +60,6 @@ parameters:
 			count: 1
 			path: app/controllers/admin/UsersController.php
 
-		-
-			message: "#^Offset 4 does not exist on null\\.$#"
-			count: 5
-			path: app/controllers/admin/UsersController.php
-
```

Comment:
Explanation: If I don't remove this, I get this error from php-lint:
![phplint](https://user-images.githubusercontent.com/43322572/135724138-da1b6478-b475-4ab6-ad32-79625f8bc84c.png)
I think it's because the ignored errors were coming from the closure functions I deleted